website/pages/docs/auth/aws.mdx

```diff
@@ -573,6 +573,28 @@ Policy](#recommended-vault-iam-policy) except it doesn't need any further
 Furthermore, in the master account, Vault must be granted the action `sts:AssumeRole`
 for the IAM Role to be assumed.
 
+### AWS Instance Metadata Timeouts
+
+~> Affects Vault 1.4 and later
+
+Anytime Vault needs to connect to the [EC2 instance metadata
+service][aws-ec2-mds] on an EC2 instance, there may be a delay. The AWS SDK used
+by Vault first attempts to connect to [v2 of the instance metadata service
+(IMDSv2)][aws-ec2-imdsv2], and if that times out, it falls back to v1. In Vault
+1.4, this timeout could take up to 2 minutes. In Vault 1.5 and later, it can
```

Comment:
I wonder if we should explicitly call out the patch version instead, i.e. 1.5.5.

Reply:
Yep, added in be0e9de